docs/governance/README.md

```diff
@@ -0,0 +1,127 @@
+For the most part, we strive to make decisions based on consensus, but as it is
+with every group of engineers, consensus can be impossible to achieve at times
+so we need to have an ultimate tie breaker. And Miguel has a lot of expertise
+and experience building .NET implementations that are supported by multiple
+parties.
+
+Please note that the member list isn't meant to be closed: as more platform
+vendors and API drivers appear, the review board will expand accordingly.
+
+## Process
+
+* **Anybody can submit proposals** for API additions to the .NET Standard.
+  Those will be tracked as issues and labeled with the [netstandard-api] label.
+* **We'll track features, no individual APIs**. In order to keep bookkeeping we
+  generally will usually file an issue per feature, rather than per API. The
+  issue description will then contain the APIs that should be added.
+* **New members on standardized types are automatically considered**. To prevent
+  accidental fragmentation, we'll automatically consider all members added by
+  any .NET implementation on types that are already in the standard. The
+  rationale here is that divergence at that the member level is not desirable
+  and unless there is something wrong with the API it's likely a good addition.
+* **Acceptance requires**
+  - A **sponsorship from a board member**. That person will be assigned the
+    issue and is expected to shepherd the issue until it's either accepted or
+    rejected. If no board member is willing to sponsor the proposal, it's
+    considered rejected.
+  - A **stable implementation** in at least one .NET implementation. The
+    implementation must be licensed under an open source license that is
+    compatible with MIT. This will allow other .NET implementations to jump-
+    start their own implementations or simply take the feature as-is.
+* **.NET Standard updates are planned** and will generally follow a set of
+  themes. We avoid grab-bag releases of .NET Standard and instead try to define
+  a set of goals that describe what kind of feature areas a particular .NET
+  Standard version provides. This simplifies answering the question which .NET
+  Standard a given library should depend on. It also makes it easier for .NET
+  implementations to decide whether it's worth implementing a higher version of
+  .NET Standard.
+* **The version number** is subject to discussion and is generally a function of
+  how significant the new version is. While we aren't planning on making
+  breaking changes, we'll rev the major version if the new version adds large
+  chunks of APIs or has sizable changes in the overall developer experience.
+
+[netstandard-api]: https://github.com/dotnet/standard/issues?q=is%3Aopen+is%3Aissue+label%3Anetstandard-api
+
+## Inclusion principles
+
+Not every .NET API needs to be (or even should be) part of the .NET Standard.
+
+If a library is written in pure IL (for example C#, VB.NET, F# etc.), and
```

Comment:
I was expecting you to say something about any such libraries APIs not needing to be included in the standard itself.

Reply:
Of course one exception to this is IL that ships in one of the platforms.

Reply:
Worth mentioning something about type forwarding?

Reply:
@weshaggard 

> I was expecting you to say something about any such libraries APIs not needing to be included in the standard itself.

That's basically what the section was about. I've slightly changed the wording to hopefully make that more clear.

> Of course one exception to this is IL that ships in one of the platforms.

Right, which is why I wrote:

> If an API ships as a library, it's much harder for any .NET implementation to use the types as part of their implementation or public API surface. Hence, there is a lot of value in adding widely used concepts to the .NET Standard in order to use them throughout all .NET implementations.